craft_cli/messages.py

```diff
@@ -402,6 +416,8 @@ class Emitter:
     the application behaviour and/or logs forensics.
     """
 
+    # pylint: disable=too-many-instance-attributes
```

Comment:
Not an issue with this PR, but:

I'm undecided whether this is a code smell (time for a minor refactor?) or an overly-sensitive lint check. If the former we might want to put in a low-priority task to do that cleanup, or if the latter maybe we should discuss with the team whether we want to disable this check (or modify the attribute count before it pops up).

Reply:
My short-ish term goal is to actually remove pylint and move to ruff (there's an issue for that), which is currently disabled because it complains about a _lot_ of things. I think we can defer that decision until then (but I would like to address that sooner rather than later).